server/tests/taxonomy_term.rs

```diff
@@ -0,0 +1,31 @@
+#[cfg(test)]
+mod set_name_and_description_mutation {
+    use test_utils::*;
+
+    #[actix_rt::test]
+    async fn sets_name_and_description() {
+        let mut transaction = begin_transaction().await;
+
+        Message::new(
+            "TaxonomyTermSetNameAndDescriptionMutation",
+            json!({
+                "id": 7,
+                "userId": 1,
+                "name": "a name",
+                "description": "a description"
+            }),
+        )
+        .execute_on(&mut transaction)
+        .await;
+
+        let query_response = Message::new("UuidQuery", json!({"id": 7}))
+            .execute_on(&mut transaction)
+            .await;
+
+        assert_ok_with(query_response, |result| {
+            assert_eq!(result["name"], "a name");
+            assert_eq!(result["description"], "a description");
+        })
+        .await;
```

Comment:
test for the new event is missing...

Reply:
done in e413673

An aside: when testing locally we see this result
```
user@serlo:~ yarn fetch EventsQuery '{ "first": 5, "objectId": 7 }'
...
    {
      "__typename": "SetTaxonomyTermNotificationEvent",
      "id": 86598,
      "instance": "de",
      "date": "2022-03-28T12:12:19+02:00",
      "actorId": 1,
      "objectId": 7,
      "taxonomyTermId": 7
    },
    {
      "__typename": "RemoveTaxonomyLinkNotificationEvent",
      "id": 76642,
      "instance": "de",
      "date": "2014-09-16T09:26:49+02:00",
      "actorId": 27687,
      "objectId": 7,
      "parentId": 7,
      "childId": 18758
    }

```
 Observe objectId 7 and parentId 7 (as well as objectId 7 and taxonomyTermId 7). Should it be so?